os/board/rtl8730e/src/component/mbed/targets/hal/rtl8730e/i2s_api.c

```diff
@@ -345,13 +290,16 @@ void i2s_set_dma_buffer(i2s_t *obj, char *tx_buf, char *rx_buf,
   */
 void i2s_tx_irq_handler(i2s_t *obj, i2s_irq_handler handler, uint32_t id)
 {
-	(void) obj;
 	(void) handler;
```

Comment:
```handler``` is being discarded here, it should be added to user callbacks and should be called in the ```i2s_tx_isr```.